src/test/java/edu/kis/powp/jobs2d/events/SelectTransformation2Command.java

```diff
@@ -0,0 +1,52 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.command.CompoundCommand;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+import edu.kis.powp.jobs2d.command.Transformations.VerticalMirrorTransformation;
+
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SelectTransformation2Command implements ActionListener {
+    private DriverManager driverManager;
+
+    public SelectTransformation2Command(DriverManager driverManager) {
+        this.driverManager = driverManager;
+    }
+
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        List<DriverCommand> commands = new ArrayList<>();
+        commands.add(new SetPositionCommand(-20, -50));
+        commands.add(new OperateToCommand(-20, -50));
+        commands.add(new SetPositionCommand(-20, -40));
+        commands.add(new OperateToCommand(-20, 50));
+        commands.add(new SetPositionCommand(0, -50));
+        commands.add(new OperateToCommand(0, -50));
+        commands.add(new SetPositionCommand(0, -40));
+        commands.add(new OperateToCommand(0, 50));
+        commands.add(new SetPositionCommand(70, -50));
+        commands.add(new OperateToCommand(20, -50));
+        commands.add(new OperateToCommand(20, 0));
+        commands.add(new OperateToCommand(70, 0));
+        commands.add(new OperateToCommand(70, 50));
+        commands.add(new OperateToCommand(20, 50));
+
+
+        VerticalMirrorTransformation transformation = new VerticalMirrorTransformation();
+        CompoundCommand compoundCommand = transformation.performTransformation(new CompoundCommand(commands));
+
+        compoundCommand.execute(driverManager.getCurrentDriver());
```

Comment:
After the transformation you should store command in DriverCommandManager (setCurrentCommand), without execution (driverManager is not necessary here). Execution can be done using runCommand menu option.